### ISSUE TITLE
Add ability to re-generate the entire changelog

### ISSUE THREAD

**Issue by MarcoIeni** [MarcoIeni/release-plz#1009]
## Motivations

Release-plz only appends the new changes to the changelog.
It would be nice if it could generate the whole changelog from scratch. This can be useful because:
- if you change the changelog configuration, you can apply the changes also to previous releases
- if you haven't used release-plz before, you can generate the changelog for previous versions

## Solution

This should be a new release-plz subcommand. `release-plz update` should only append the changes for the new version.

TODO

## Additional context

Suggested [here](https://github.com/MarcoIeni/release-plz/discussions/1002#discussioncomment-7151904)

**Comment by intgr** [comment]
You can also call `git cliff` directly. By default it will generate the whole changelog.

**Comment by MarcoIeni** [comment]
yes, we just need to make sure to use the same release-plz config:
https://github.com/MarcoIeni/release-plz/blob/c5eed2fb325420582b8ad7ede4ccbae74fa69fd9/crates/release_plz_core/src/changelog.rs#L225

I need to document this better
